mobile/library/java/io/envoyproxy/envoymobile/engine/EnvoyEngineImpl.java

```diff
@@ -120,7 +120,7 @@ public EnvoyStatus runWithConfig(EnvoyConfiguration envoyConfiguration, String l
         return EnvoyStatus.ENVOY_SUCCESS;
       }
     } catch (Throwable throwable) {
-      // TODO: Need to have a way to log the exception somewhere.
+      throw throwable;
```

Comment:
Instead of catching and re-throwing, would it make sense to just not catch at all? (It's been a while since I've done much Java, so this might be a bad suggestion!)